docs/manual/external-tools.tex

```diff
@@ -960,12 +962,34 @@ \subsection{Adding a checker via the project properties\label{netbeans-project-p
 \end{enumerate}
 
 
+\subsection {Adding a checker using the Checker Framework Plugin\label{netbeans-plugin}}
+
+In order to run a checker using the NetBeans plugin, you should first
+install the plugin. To do so, select ``Tools'' and then ``Plugins''.
+In the Plugins window, click on the ``Settings'' tab, and then ``Add''.
+In the popup that appears, enter ``Checker Framework'' under ``Name'' and
+the URL where the Checker Framework plugin is hosted under ``URL''.
+
+Currently, the Checker Framework plugin is kept at \url{https://twlogger.github.io/catalog.xml}.
+
+After adding the Checker Framework plugin Update Center, go to the
+``Available Plugins'' tab, and search for ``Checker Framework'', select
+the checkbox, and click ``Install''.
+
+Once the Checker Framework is installed, right click on the project
+that you wish to run the checker on, and click ``Properties''. Then,
+select the checkboxes of the checkers that you wish to run, and press
+``Ok'' to apply the changes. If ``Compile on save'' is enabled (this can
+also be set in the Project Properties window), the results of the checker
+should appear automatically in the editor.
```

Comment:
When I follow the instructions I provided in Netbeans 8,1, I can get the checker output to appear in the editor with no issue, but I got the same results as you when I tried them on Netbeans 8.2.

I had not tested on Netbeans 8.2 before, so I did not notice that the checker output does not appear in the editor in 8.2. 

I'll investigate why this is happening and correct it. For now, I can only say for certain that the plugin will work on Netbeans 8.1.